packages/angular_devkit/build_optimizer/src/build-optimizer/build-optimizer.ts

```diff
@@ -54,6 +54,11 @@ function isKnownCoreFile(filePath: string) {
 }
 
 function isKnownSideEffectFree(filePath: string) {
+  // rxjs add imports contain intentional side effects
+  if (/[\\/]node_modules[\\/]rxjs[\\/]add[\\/]/.test(filePath)) {
```

Comment:
Wasn't this pattern used in RxJs 5, which is no longer supported.

Reply:
Still usable in 6.x and worked until the recent fix for the size regression.
Once 7 is the minimum (and released) this logic here and in the webpack configuration can be removed.